### PR TITLE
[ADVAPP-781]: Enhance the Prospect and Student care team experience by allowing the ability to add more than one staff member at a time

### PR DESCRIPTION
**Pull request by ketan-canyon**
### Ticket(s) or GitHub Issue

- https://canyongbs.atlassian.net/browse/ADVAPP-781

### Technical Description

Enhance the Prospect and Student care team experience by allowing the ability to add more than one staff member at a time

### Any deployment steps required?

No

### Are any Feature Flags Added?

No
_______________________________________________

### CODE REVIEW

File: app-modules/student-data-model/src/Filament/Resources/StudentResource/Pages/ManageStudentCareTeam.php [hunk 1]
```diff
@@ -92,15 +92,21 @@ public function table(Table $table): Table
                     ->attachAnother(false)
                     ->color('primary')
                     ->recordSelect(
-                        fn (Select $select) => $select->placeholder('Select a User'),
+                        fn (Select $select) => $select->placeholder('Select a Users'),
```
Comment (Orrison) [inline app-modules/student-data-model/src/Filament/Resources/StudentResource/Pages/ManageStudentCareTeam.php:95]:
```suggestion
                        fn (Select $select) => $select->placeholder('Select Users'),
```

File: app-modules/prospect/src/Filament/Resources/ProspectResource/Pages/ManageProspectCareTeam.php [hunk 2]
```diff
@@ -94,13 +94,19 @@ public function table(Table $table): Table
                     ->recordSelect(
                         fn (Select $select) => $select->placeholder('Select a User'),
```
Comment (Orrison) [inline app-modules/prospect/src/Filament/Resources/ProspectResource/Pages/ManageProspectCareTeam.php:95]:
```suggestion
                        fn (Select $select) => $select->placeholder('Select Users'),
```

**Review by Orrison — CHANGES_REQUESTED** [review]
Looks great! Just a couple of grammatical changes are needed.

File: app-modules/student-data-model/src/Filament/Resources/StudentResource/Pages/ManageStudentCareTeam.php (hunk 1 above)
Comment (ketan-canyon) [reply app-modules/student-data-model/src/Filament/Resources/StudentResource/Pages/ManageStudentCareTeam.php]: Done

File: app-modules/prospect/src/Filament/Resources/ProspectResource/Pages/ManageProspectCareTeam.php (hunk 2 above)
Comment (ketan-canyon) [reply app-modules/prospect/src/Filament/Resources/ProspectResource/Pages/ManageProspectCareTeam.php]: Done

**Review by Orrison — APPROVED** [review]

**Pull request merged**